app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/artist/ArtistAdapter.java

```diff
@@ -0,0 +1,74 @@
+package com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.artist;
+
+import android.content.Context;
+import android.graphics.Bitmap;
```

Comment:
ERROR: Unused import: android.graphics.Bitmap. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports